Common/Platform/x86x64/Linux/PlatformDefines.h

```diff
@@ -37,6 +37,50 @@ using tchar = char;
 using tstring = std::string;
 #endif
 
+#ifdef __linux__
+#define WINAPIV
+#elif _WIN32
+#define WINAPIV __cdecl
```

Comment:
Why do you know WINAPIV is __cdecl in WIN32?
LINUX not linux
Also, in my previous comment, it should be OS_API:
#ifdef __LINUX__
#define OS_API
#endif __WINDOWS__
#define OS_API WINAPIV
#endif